charts/memgraph/templates/service.yaml

```diff
@@ -15,5 +15,12 @@ spec:
       targetPort: {{ .Values.service.targetPort}}
       protocol: {{ .Values.service.protocol }}
       name: bolt
-  selector:
-    {{- include "memgraph.selectorLabels" . | nindent 4 }}
+    - port: 7444
+      protocol: {{ .Values.service.protocol }}
+      name: websocket
+    - port: 9091
```

Comment:
Can we move the actual port value (`7444`, `9091`) to `values.yaml` file and add something like `monitoring-websocket` and `monitoring-http`? 

To make it more exact what is being used for.

Reply:
Yes this will give more control on the ports assignments.